### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by sfingerhut**
**What this PR does / why we need it**:
Added language for people to start with Grafana Cloud

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
@sfingerhut thank you for your contribution. However, this is not the correct Getting Started topic to update. It is for on-prem Grafana. We do link to Grafana Cloud documentation from here. That would be the appropriate section to make the changes.
Hope this makes sense.

**Comment by sfingerhut** [comment]
A lot of people start here regardless of intention - they end up here through Google search, not necessarily intending for only on-prem. I see no harm at all to mention and link to Grafana Cloud - we can send them to docs in Grafana Cloud as well.

**Comment by achatterjee-grafana** [comment]
We can definitely add a separate section, maybe in the introduction. But I wouldn't update the existing content. Is that an acceptable solution?

**Comment by sfingerhut** [comment]
I believe this specific page (not another section) and any page that makes sense should reference Grafana Cloud. This especially needs to be here. Let me give the example of Elastic - where we merged all the Cloud stuff across. This had an immediate positive impact on Cloud adoption and business. 
https://www.elastic.co/guide/en/elastic-stack/current/installing-elastic-stack.html

**Comment by achatterjee-grafana** [comment]
Sounds like we are on the same page!

File: docs/sources/getting-started/getting-started.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ weight = 200
 
 This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "_index.md" >}}).
```
Comment (achatterjee-grafana) [inline docs/sources/getting-started/getting-started.md:11]:
```suggestion
This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "_index.md" >}}).

Grafana also offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) (10K metrics, 50 GB logs, and dashboards) to make starting even easier and faster. You can install Grafana to self-host or get a free Grafana Cloud account.
```

File: docs/sources/getting-started/getting-started.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ weight = 200
...
 
-## Step 1: Install Grafana
+## Step 1: Install Grafana to self host or get a Free Grafana Cloud Account
```
Comment (achatterjee-grafana) [inline docs/sources/getting-started/getting-started.md:13]:
```suggestion
## Step 1: Install Grafana. 
```

File: docs/sources/getting-started/getting-started.md [hunk 3]
```diff
@@ -10,9 +10,9 @@ weight = 200
 
 This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "_index.md" >}}).
 
-## Step 1: Install Grafana
+## Step 1: Install Grafana to self host or get a Free Grafana Cloud Account 
 
-Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).
+Grafana offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) (10K metrics, 50gb logs and dashboards) to make starting even easier and faster. Alternatively, Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).
```
Comment (achatterjee-grafana) [inline docs/sources/getting-started/getting-started.md:15]:
```suggestion
Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Added review comments.

File: docs/sources/getting-started/getting-started.md (hunk 3 above)
Comment (osg-grafana) [inline docs/sources/getting-started/getting-started.md:15]: +1 to not include plan-specific details in a Getting started document.

This causes a maintenance issue where those numbers are subject to change without timely cross-team knowledge of such an information dependency, and as a result user become confused when they sees specifics in one area that do match specifics in another area.

**Review by osg-grafana — CHANGES_REQUESTED** [review]

File: docs/sources/getting-started/getting-started.md (hunk 3 above)
Comment (sfingerhut) [reply docs/sources/getting-started/getting-started.md:15]: We can kill the reference to 10K metrics, 50gb logs and dashboards, but I want the Grafana Cloud with link to a signup in there.

Comment (achatterjee-grafana) [reply docs/sources/getting-started/getting-started.md:15]: Agree with @osg-grafana. This content should really be in the topic "What is Grafana?" but I based my review keeping in mind @sfingerhut's past experiences.

File: docs/sources/getting-started/getting-started.md (hunk 2 above)
Comment (osg-grafana) [reply docs/sources/getting-started/getting-started.md:13]: +1

**Review by achatterjee-grafana — COMMENTED** [review]
Updated my review based on our conversation.

**Comment by sfingerhut** [comment]
Please push this update or let me know what would block it.

**Review by osg-grafana — APPROVED** [review]
Thank you both. Have a good [time of day here].

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

File: docs/sources/getting-started/getting-started.md [hunk 4]
```diff
@@ -10,9 +10,11 @@ weight = 200
 
 This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "_index.md" >}}).
 
-## Step 1: Install Grafana
+> **Note:** Grafana also offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) to help getting starting even easier and faster. You can install Grafana to self-host or get a free Grafana Cloud account.
```
Comment (osg-grafana) [inline docs/sources/getting-started/getting-started.md]:
```suggestion
> **Note:** Grafana also offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) to help getting started even easier and faster. You can install Grafana to self-host or get a free Grafana Cloud account.
```

**Review by osg-grafana — CHANGES_REQUESTED** [review]
-ing --> -ed

**Review by osg-grafana — APPROVED** [review]

File: docs/sources/getting-started/getting-started.md [hunk 5]
```diff
@@ -10,9 +10,11 @@ weight = 200
 
 This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "_index.md" >}}).
 
-## Step 1: Install Grafana
+> **Note:** Grafana also offers a [free account with Grafana Cloud](https://grafana.com/signup/cloud/connect-account?pg=gsdocs) to help getting started even easier and faster. You can install Grafana to self-host or get a free Grafana Cloud account.
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started.md:13]: @matthewhelmke, should we link to the quickstart topic from here?

File: docs/sources/getting-started/getting-started.md [hunk 6]
```diff
@@ -10,9 +10,11 @@ weight = 200
...
 
-Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).
+## Step 1: Install Grafana.
```
Comment (oddlittlebird) [inline docs/sources/getting-started/getting-started.md:15]:
```suggestion
## Step 1: Install Grafana
```

**Review by oddlittlebird — APPROVED** [review]
Good idea, @sfingerhut. Please let @grafana/docs-squad or @matthewhelmke know if you find other places that should be connected.

**Comment by sfingerhut** [comment]
I don't see a change on the page yet. What is blocking this?

**Comment by achatterjee-grafana** [comment]
This is still open. I will check all changes are in and merge this. I will let you know when it is done. Thanks!

**Comment by sfingerhut** [comment]
Just a quick share - we've already seen 34 signups from this link! thanks all.

**Pull request merged**